### ISSUE TITLE
Chaining profilers

### ISSUE THREAD

**Issue by bobuva** [dotnet/coreclr#16366]
In years past, I've spoken with various people at Microsoft who were interested in running multiple profilers  (that use the .NET Profiler API) on the same machine. Has any consideration been given to this for the now open-source profiler here in coreclr?

**Comment by RussKeldorph** [comment]
@noahfalk

**Comment by lt72** [comment]
@bobuva: yes, we have considered the issue, but we do not believe we will proceed to design/implementation any time soon. Although this is definitely on our TODO list, we did not have many concrete requests on this subject yet.

**Comment by bobuva** [comment]
@lt72 Thanks for the reply. Although you haven't seen many requests, there are in fact products on the market that use the profiler API. Attempting to have another profiler api-based tool reside side-by-side with one of them is currently impossible. I would be interested in continuing to speak to whomever is keeping an eye on that item.

**Comment by lt72** [comment]
That would me either myself or @noahfalk

**Comment by lt72** [comment]
We are not participating directly, but it will be a pleasure to stop by and chat! I'll send you my contact so you can ping me directly when the time comes and we can agree on details then.

**Comment by bobuva** [comment]
Thanks!
